sdk/attestation/azure-security-attestation/samples/README.md

```diff
@@ -0,0 +1,176 @@
+
+## Sample Requirements
+
+These samples are written with the assumption that the following environment
+variables have been set by the user:
+
+* ATTESTATION_AAD_URL - the base URL for an attestation service instance in AAD mode.
+* ATTESTATION_ISOLATED_URL - the base URL for an attestation service instance in Isolated mode.
+* ATTESTATION_LOCATION_SHORT_NAME - the short name for the region in which the
+    sample should be run - used to interact with the shared endpoint for that
+    region.
+* ATTESTATION_ISOLATED_SIGNING_CERTIFICATE - The DER encoded form of the signing
+    certificate used to create an isolated attestation instance, Base64 encoded.
+* ATTESTATION_ISOLATED_SIGNING_KEY - The DER encoded of an RSA Private key, which
+    was used to create an isolated attestation service instance.
+
+The tests also assume that the currently logged on user is authorized to call
+into the attestation service instance because they use [DefaultAzureCredential](https://docs.microsoft.com/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python) for authorization.
+
+## Samples descriptions
+
+**File Name** | **Description**
+|-----|-------|
+| [sample_authentication.py](https://aka.ms/azsdk/guideline/links) and [sample_authentication_async.py](https://aka.ms/azsdk/guideline/links) | Authenticate a connection with the attestation service.|
+| [sample_attest_enclave.py](https://aka.ms/azsdk/guideline/links) and [sample_attest_enclave_async.py](https://aka.ms/azsdk/guideline/links) | Attest an SGX and OpenEnclave enclave with the attestation service. Also shows how to use the TokenValidationOptions to perform additional validation of the returned attestation token and validation keys.|
+|[sample_get_set_policy.py](https://aka.ms/azsdk/guideline/links) and [sample_get_set_policy_async.py](https://aka.ms/azsdk/guideline/links) | Policy manipulation operations - Get, Set, Reset on different attestation types.|
+
+### Prerequisites
+
+* Python 2.7, or 3.6 or later is required to use this package.
+* You must have an [Azure subscription](https://azure.microsoft.com/free/) and either an
+[Azure storage account](https://docs.microsoft.com/azure/storage/common/storage-account-overview) or an [Azure Cosmos Account](https://docs.microsoft.com/azure/cosmos-db/account-overview) to use this package.
+
+## Setup
+
+1. Install the Azure Attestation client library for Python with [pip](https://pypi.org/project/pip/):
+
+```bash
+pip install --pre azure-security-attestation
+```
+
+2. Clone or download this sample repository
+1. Open the sample folder in Visual Studio Code or your IDE of choice.
+
+## Running the samples
+
+1. Open a terminal window and `cd` to the directory that the samples are saved in.
+1. Set the environment variables specified in the sample file you wish to run.
+1. Follow the usage described in the file, e.g. `python sample_attest_enclave.py`
```

Comment:
1. , 2. , 3.

Reply:
QQ: Is it not considered best practice to let Markdown do the enumeration for me? Or is it preferred to specifically enumerate the entries in an enumeration?

Reply:
I don't think we have a best practice here (not to my knowledge), but having the list in the proper order is definitely preferred.

Reply:
I've confirmed that the rich diff for this file shows the enumeration correctly ordered.